core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1113,11 +1116,12 @@ class ReplicaManager(val config: KafkaConfig,
         } catch {
           case e: KafkaStorageException =>
             stateChangeLogger.error(("Broker %d skipped the become-follower state change with correlation id %d from " +
-              "controller %d epoch %d for partition [%s,%d] since the replica for the partition is offline due to disk error %s")
-              .format(localBrokerId, correlationId, controllerId, partitionStateInfo.basePartitionState.controllerEpoch, partition.topic, partition.partitionId, e))
-            val dirOpt = getLogDir(new TopicPartition(partition.topic, partition.partitionId))
+              "controller %d epoch %d for partition %s since the replica for the partition is offline due to disk error %s")
+              .format(localBrokerId, correlationId, controllerId, partitionStateInfo.basePartitionState.controllerEpoch,
+                partition.topicPartition, e))
```

Comment:
String interpolation here too?

Reply:
ugh. you knew that mentioning this would force me to change all the others too